teams/wg-embedded-tools.toml

```diff
@@ -0,0 +1,20 @@
+name = "wg-embedded-tools"
+subteam-of = "wg-embedded"
+kind = "working-group"
+
+[people]
+leads = []
+members = [
+    "almindor",
+    "Disasm",
+    "thejpster",
+    "dkhayes117",
+]
```

Comment:
This member list seems wrong

Reply:
huh, good catch, thanks! i must have got addled somewhere in the manual transcription.